src/pubsub/handlers/setApiResponseHandler.js

```diff
@@ -102,4 +104,14 @@ export default class SetApiResponseHandler extends BaseHandler {
     const reportIdString = JSON.stringify({ report: 'Received UserInterest apiResponse parameters' });
     return reportIdString;
   }
+
+  // importing external Api resonse function, which can set the pre-requisite values to external modules
+  setExternalResponse(message) {
+    try {
+      const externalFunction = require('../../../plugins/setExternalApiResponse');
```

Comment:
Is it okay to give reference of plugin here? @neeradanelxsi

Reply:
We should be okay to reference plugins as we do in other code but the `require` does make wonder if that will cause issues? Either way we should have explain in some doc how this would be setup for OS users.

Reply:
Ideally this change should be in **_fca-parent_** setApiResponseHandler

Reply:
A seperate section has been added in readme on steps to integrate external sdk and details regarding the same has been added there. This code was added by Guru and I think the reasoning was to reuse the existing handler in FCA for setting response instead of creating a new one